src/main/java/org/springframework/data/redis/core/DefaultSetOperations.java

```diff
@@ -30,6 +30,7 @@
  * @author Costin Leau
  * @author Christoph Strobl
  * @author Mark Paluch
+ * @author Dominys
```

Comment:
Thanks for adding `@author` tags. Please use your actual name, not the GitHub handle or nickname.

Reply:
@mp911de Ok. I'll change this.